src/pybind/mgr/cephadm/services/ingress.py

```diff
@@ -20,6 +20,16 @@ class IngressService(CephService):
     TYPE = 'ingress'
     MAX_KEEPALIVED_PASS_LEN = 8
 
+    @staticmethod
+    def get_dependencies(mgr: "CephadmOrchestrator",
```

Comment:
This could be a classmethod and it would behave "correctly" wrt inheritance.

Reply:
same comment as above :)